core/cli/cli.js

```diff
@@ -54,17 +55,18 @@ function start() {
 
     const reframeConfig = reframeConfigPath && require(reframeConfigPath);
 
-    startHapiServer({pagesDirPath, reframeConfig, appDirPath});
+    startHapiServer({pagesDirPath, reframeConfig, appDirPath}, showHapiServerLog);
```

Comment:
Or we just use the argument object: `{pagesDirPath, reframeConfig, appDirPath, showHapiServerLog}`. I like arg objects because I know that both ends, the function body and the function caller, will use the same variable names.